src/main/resources/view/MainWindow.fxml

```diff
@@ -6,15 +6,17 @@
 <?import javafx.scene.control.Menu?>
 <?import javafx.scene.control.MenuBar?>
 <?import javafx.scene.control.MenuItem?>
-<?import javafx.scene.control.SplitPane?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.image.ImageView?>
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
-         title="AssetBook-3" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="AssetBook" minWidth="600" minHeight="800" onCloseRequest="#handleExit">
```

Comment:
Would it be better to change minHeight to 700? The minHeight of 800 causes the app to go out of my screen, and forces the window to be unable to become smaller.

Reply:
Min width and height reverted